proto/api/read_state.proto

```diff
@@ -25,6 +25,7 @@ message ReadStakingDataMethod {
 		CANDIDATES = 4;
 		CANDIDATE_BY_NAME = 5;
 		BUCKETS_BY_INDEXES = 6;
+		CANDIDATE_BY_ADDRESS = 7;
```

Comment:
need by operator and by reward addr?

Reply:
owner address

Reply:
oh, nvm, it seems by owner address search only